lib/controller/FotoAlbumController.class.php

```diff
@@ -374,4 +381,24 @@ public function removetag() {
 		}
 	}
 
+	public function raw_image($path) {
+		//Extra check to prevent attacks
+		if (!startsWith(realpath($path), realpath(PHOTOS_PATH))) {
+			$this->exit_http(403);
+		}
+
+		$image = Foto::fromFileName($path);
+		if ($image === false || !$image->magBekijken()) {
+			$this->exit_http(403);
+		} else if (!$image->exists()) {
+			$this->exit_http(403);
+		}
+
+		$fp = fopen($image->getFullPath(), 'rb');
```

Comment:
Codacy vindt `$fp` een te korte variabele

Reply:
Ja, dat had ik gezien. Wordt gefixt in mijn volgende commit